py7zr/py7zr.py

```diff
@@ -1430,12 +1434,19 @@ def decompress(
         max_block_size = get_memory_limit()
         crc32 = 0
         decompressor = folder.get_decompressor(compressed_size)
+        report_step_size = 25 * 2 ** 20
```

Comment:
Please don't use direct value. Could you define constant with a reason of numbers as comment?

Reply:
Alternatively, a parameter to ExtractCallback that gets passed down would be appreciated.

Reply:
I've found out another approach.